dem_stitcher/stitcher.py

```diff
@@ -225,22 +164,42 @@ def patch_glo_30_with_glo_90(arr_glo_30: np.ndarray,
     stitcher_kwargs['dem_name'] = 'glo_90_missing'
     arr_glo_90, prof_glo_90 = stitch_dem(**stitcher_kwargs)
 
-    dem_arr, dem_prof = merge_glo_30_and_90_dems(arr_glo_30,
-                                                 prof_glo_30,
-                                                 arr_glo_90,
-                                                 prof_glo_90
-                                                 )
+    dem_arr, dem_prof = merge_arrays_with_geometadata([arr_glo_30, arr_glo_90],
+                                                      [prof_glo_30, prof_glo_90]
+                                                      )
 
     return dem_arr, dem_prof
 
 
+def _translate_one_tile_across_dateline(dataset: rasterio.DatasetReader, crossing):
+
+    assert crossing in [180, -180]
+    res_x = dataset.res[0]
+    xmin, _, xmax, _ = dataset.bounds
+
+    tags = dataset.tags()
+    if crossing == 180 and xmax <= 0:
+        memfile, dataset_new = translate_dataset(dataset, 360 / res_x, 0)
+        # Ensures area or point are correctly stored!
+        dataset_new.update_tags(**tags)
+    elif crossing == -180 and xmin >= 0:
+        memfile, dataset_new = translate_dataset(dataset, -360 / res_x, 0)
+        # Ensures area or point are correctly stored!
+        dataset_new.update_tags(**tags)
```

Comment:
Copying tags from the tiles ensures rest of workflow is correct. If not tags are present, the tiles are assumed to have `AREA` tag.